hack/jenkins/e2e.sh

```diff
@@ -221,462 +123,333 @@ case ${JOB_NAME} in
-    : ${E2E_NETWORK:="e2e-enormous-cluster"}
-    : ${E2E_TEST:="false"}
-    : ${KUBE_GCE_INSTANCE_PREFIX:="e2e-enormous-startup"}
-    : ${PROJECT:="kubernetes-scale"}
+    export E2E_NAME="e2e-enormous-cluster"
+    export E2E_TEST="false"
+    export PROJECT="kubernetes-scale"
+    export FAIL_ON_GCP_RESOURCE_LEAK="false"
     # Override GCE defaults.
-    NETWORK_PROVIDER="flannel"
+    export NETWORK_PROVIDER="flannel"
     # Temporarily switch of Heapster, as this will not schedule anywhere.
     # TODO: Think of a solution to enable it.
-    ENABLE_CLUSTER_MONITORING="none"
-    E2E_ZONE="us-east1-b"
-    MASTER_SIZE="n1-standard-32"
-    NODE_SIZE="n1-standard-1"
-    NODE_DISK_SIZE="50GB"
-    NUM_NODES="1000"
+    export KUBE_ENABLE_CLUSTER_MONITORING="none"
+    export KUBE_GCE_ZONE="us-east1-b"
+    export MASTER_SIZE="n1-standard-32"
+    export NODE_SIZE="n1-standard-1"
+    export NODE_DISK_SIZE="50GB"
+    export NUM_NODES="1000"
     # Reduce logs verbosity
-    TEST_CLUSTER_LOG_LEVEL="--v=1"
+    export TEST_CLUSTER_LOG_LEVEL="--v=1"
     # Increase resync period to simulate production
-    TEST_CLUSTER_RESYNC_PERIOD="--min-resync-period=12h"
+    export TEST_CLUSTER_RESYNC_PERIOD="--min-resync-period=12h"
     ;;
 
   # Run Kubemark test on a fake 100 node cluster to have a comparison
   # to the real results from scalability suite
   kubernetes-kubemark-gce)
-    : ${E2E_CLUSTER_NAME:="kubernetes-kubemark"}
-    : ${E2E_NETWORK:="kubernetes-kubemark"}
-    : ${PROJECT:="k8s-jenkins-kubemark"}
-    : ${E2E_UP:="true"}
-    : ${E2E_DOWN:="true"}
-    : ${E2E_TEST:="false"}
-    : ${USE_KUBEMARK:="true"}
-    : ${KUBEMARK_TESTS:="\[Feature:Performance\]"}
+    export PROJECT="k8s-jenkins-kubemark"
+    export E2E_TEST="false"
+    export USE_KUBEMARK="true"
+    export KUBEMARK_TESTS="\[Feature:Performance\]"
     # Override defaults to be indpendent from GCE defaults and set kubemark parameters
-    KUBE_GCE_INSTANCE_PREFIX="kubemark100"
-    NUM_NODES="10"
-    MASTER_SIZE="n1-standard-2"
-    NODE_SIZE="n1-standard-1"
-    E2E_ZONE="us-central1-b"
-    KUBEMARK_MASTER_SIZE="n1-standard-4"
-    KUBEMARK_NUM_NODES="100"
+    export NUM_NODES="10"
+    export MASTER_SIZE="n1-standard-2"
+    export NODE_SIZE="n1-standard-1"
+    export KUBE_GCE_ZONE="us-central1-b"
+    export KUBEMARK_MASTER_SIZE="n1-standard-4"
+    export KUBEMARK_NUM_NODES="100"
     ;;
 
   # Run Kubemark test on a fake 500 node cluster to test for regressions on
   # bigger clusters
   kubernetes-kubemark-500-gce)
-    : ${E2E_CLUSTER_NAME:="kubernetes-kubemark-500"}
-    : ${E2E_NETWORK:="kubernetes-kubemark-500"}
-    : ${PROJECT:="kubernetes-scale"}
-    : ${E2E_UP:="true"}
-    : ${E2E_DOWN:="true"}
-    : ${E2E_TEST:="false"}
-    : ${USE_KUBEMARK:="true"}
-    : ${KUBEMARK_TESTS:="\[Feature:Performance\]"}
+    # XXX Not a unique project
+    export E2E_NAME="kubemark-500"
```

Comment:
@ihmccreery - you completely broke kubemark-500 and kubemark-scale
since you are not setting E2E_NETWORK (which was set)

Can you please check it next time?

@gmarek